src/manipulation.js

```diff
@@ -46,14 +46,14 @@ wrapMap.optgroup = wrapMap.option;
 wrapMap.tbody = wrapMap.tfoot = wrapMap.colgroup = wrapMap.caption = wrapMap.thead;
 wrapMap.th = wrapMap.td;
 
-// Manipulating tables requires a tbody
 function manipulationTarget( elem, content ) {
-	return jQuery.nodeName( elem, "table" ) &&
-		jQuery.nodeName( content.nodeType !== 11 ? content : content.firstChild, "tr" ) ?
+	if ( jQuery.nodeName( elem, "table" ) &&
+		jQuery.nodeName( content.nodeType !== 11 ? content : content.firstChild, "tr" ) ) {
 
-		elem.getElementsByTagName("tbody")[0] ||
-			elem.appendChild( elem.ownerDocument.createElement("tbody") ) :
-		elem;
+		return elem.getElementsByTagName( "tbody" )[ 0 ] || elem;
```

Comment:
In the old code, we guaranteed that a `<tr>` never got appended to a `<table>`. Do all the supported browsers in master auto-insert the `<tbody>` for us now? Do we have a unit test verifying that? I see some weak existing tests [here](https://github.com/markelog/jquery/blob/fdd04a4d47bc599b2615ec2eda01d19d469e24c5/test/unit/manipulation.js#L262-271) and [here](https://github.com/markelog/jquery/blob/fdd04a4d47bc599b2615ec2eda01d19d469e24c5/test/unit/manipulation.js#L1364-1377) but don't think they are making sure of this.